boards/arm/96b_carbon_nrf51/96b_carbon_nrf51.yaml

```diff
@@ -9,3 +9,6 @@ toolchain:
   - gccarmemb
 supported:
   - ble
+testing:
+  ignore_tags:
+    - net
```

Comment:
btw, most of these boards with 16kb ram were not building any net samples because of sanitycheck script bug fixed in this set, with this bug fixed, most of them fail on these boards.